katello.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |gem|
   gem.homepage    = "http://www.katello.org"
   gem.summary     = "Content and Subscription Management plugin for Foreman"
   gem.description = "Katello adds Content and Subscription Management to Foreman. For this it relies on Candlepin and Pulp."
-  gem.required_ruby_version = '~> 2.5'
+  gem.required_ruby_version = '>= 2.7'
```

Comment:
Should this get an upper bound? In https://github.com/Katello/katello/pull/10861/files#diff-21cb8426e694930953a1fb66d2f1e4c0c4021bcadd760d9582423094a4dcaaceR16 you specificed `< 3.1`, in other plugins we used `< 4`.